backend/src/src-fossology/src/main/java/org/eclipse/sw360/fossology/FossologyHandler.java

```diff
@@ -475,4 +475,38 @@ private String createReportAttachmentName(Release release) {
         return release.getName() + "-" + release.getVersion() + "-" + dateTimeFormatter.format(now) + "-SPDX.rdf";
     }
 
+    @Override
+    public RequestStatus triggerReportGenerationFossology(String releaseId, User user) throws TException {
```

Comment:
Please check if `throws TException ` can be removed because it is already handled by `try - catch`

Reply:
Removed the try-catch and handled it in caller method.